### DISCUSSION TITLE
need a refund

### DISCUSSION THREAD

**Discussion by mohmedx20** [community/community#41820] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

There was an auto-renewal payment on copilot and I did not notice now I got charged $10 I canceled the subscription and I want a refund please it used my PayPal account.

**Comment by KDA809** [comment]
its probably best and quicker to go through the Github help https://support.github.com/ and create ticket :)

**Reply by bchellingworth** [reply]
its $10, surely its more cost effective to just get on with something else...

**Reply by mohmedx20** [reply]
Well you just don’t know how might $10 be in my country be right so it might be alot for me and will be good to have it back

**Comment by Akash1134** [comment, marked as answer]
Hi Everyone,

We now have the option to self-serve refunds for Copilot. Please head over to the [Help with your GitHub account](https://support.github.com/contact/account?flow=refund_copilot_individual) section and save yourself some time.

Note that, in most cases, the Virtual Agent may be able to offer an immediate refund. But, if your concern is not resolved, you can always use the contact form to get in touch with our billing team.

**Comment by Squad-Root-Developers** [comment]
Prezados!

Contratei hoje dia 23/05/2024 o GitHub Team com 2 membros por $8.00 pelos dois membros. No momento de escolher o membro acabei colocando um membro a mais e fui cobrado mais $4.00 referente a mais um membro. Removi o membro inserido por engano e agora preciso que o valor seja reembolsado pois a inclusão do 3º menbro não foi intencional.

Atenciosamente.

![cobranca-github-23-05-2024](https://github.com/community/community/assets/170661135/1220ec9d-b5a9-4512-820e-ae058c9ccf76)

**Comment by m4rilynv** [comment]
I been trying get refund for a while I never tesolution

**Comment by m4rilynv** [comment]
On multiple charges unauthorized
